### DISCUSSION TITLE
What does "Allow other apps to access my data" toggle do?

### DISCUSSION THREAD

**Discussion by edarzh** [CatimaLoyalty/Android#1624] [Q&A] [answered]
The option is self explanatory, I just don't know where I could use this

**Comment by TheLastProject** [comment, marked as answer]
Currently it's only used by [Gadgetbridge](https://gadgetbridge.org/) to make Catima cards available on Bangle.js and select ZeppOS devices.

I turned this into #1625 because I do think it's an issue in Catima to not explain this better

**Reply by edarzh** [reply]
Thanks 👍
